release/internal/version/version.go

```diff
@@ -100,17 +100,36 @@ func (v *Version) Milestone() string {
 	return fmt.Sprintf("%s v%d.%d.%d", utils.DisplayProductName(), ver.Major(), ver.Minor(), ver.Patch())
 }
 
-// Stream returns the "release stream" of the version, i.e., the major and minor version without the patch version.
+// Stream returns the "release stream" of the version
+// typically it is the major and minor version without the patch version.
+// For example, for version "v3.15.0", the stream is "v3.15".
+// For EP 1, appends "-1" to the stream.
 func (v *Version) Stream() string {
 	ver := semver.MustParse(string(*v))
-	return fmt.Sprintf("v%d.%d", ver.Major(), ver.Minor())
+	stream := fmt.Sprintf("v%d.%d", ver.Major(), ver.Minor())
+	if ver.Prerelease() != "" && strings.HasPrefix(ver.Prerelease(), "1.") {
```

Comment:
Should we have an explicit flag passed in to say when something is an EP release? 

My concern is that this approach (using "secret" knowledge about the syntax of a version string) is brittle and not super intuitive.

At a minimum, we should have an `IsEarlyPreviewVersion(v string) bool` function that we can reuse, but still think a flag would be ideal.